Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -0,0 +1,110 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class CommentCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  fileprivate let viewModel = CommentCellViewModel()
+
+  private lazy var rootStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var bodyTextView: UITextView = { UITextView(frame: .zero) }()
+  private lazy var replyButton = { UIButton(frame: .zero) }()
+  private lazy var flagButton = { UIButton(frame: .zero) }()
+  private lazy var commentCellHeaderStackView: CommentCellHeaderStackView = {
+    CommentCellHeaderStackView(frame: .zero)
+  }()
+
+  private lazy var bottomColumnStackView: UIStackView = { UIStackView(frame: .zero) }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.setupConstraints()
+    self.bindStyles()
+    self.configureViews()
+    self.bindViewModel()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+
+    _ = self.rootStackView
+      |> commentCellRootStackViewStyle
+
+    _ = self.bodyTextView
+      |> commentBodyTextViewStyle
+
+    _ = self.replyButton
+      |> replyButtonStyle
+
+    _ = self.flagButton
+      |> UIButton.lens.image(for: .normal) %~ { _ in Library.image(named: "flag") }
+  }
+
+  // MARK: - Configuration
+
+  internal func configureWith(value: DemoComment) {
```

Comment:
Not sure why we need `DemoComment`, the `Comment` model does exist in the codebase. If you need to access the template, just import KsApi with `@testable`. Either way let's just be sure not to merge `DemoComment` 👍